optuna_dashboard/_form_widget.py

```diff
@@ -138,6 +140,19 @@ def to_dict(self) -> UserAttrRefJSON:
 def register_objective_form_widgets(
     study: optuna.Study, widgets: list[ObjectiveFormWidget]
 ) -> None:
+    if len(study.directions) != len(widgets):
+        raise ValueError("The length of actions must be the same with the number of objectives.")
+
+    if version.parse(optuna_ver) <= version.Version("3.1.1"):
```

Comment:
```suggestion
    if version.parse(optuna_ver) < version.Version("3.2"):
```
Should we consider a patch release of 3.1, such as 3.1.2?